src/lib/features/personal-dashboard/personal-dashboard-controller.e2e.test.ts

```diff
@@ -61,3 +62,84 @@ test('should return personal dashboard with own flags and favorited flags', asyn
         ],
     });
 });
+
+const createProject = async (name: string, user: IUser) => {
+    const auditUser = {
+        id: 1,
+        username: 'audit user',
+        ip: '127.0.0.1',
+    };
+    const project = await app.services.projectService.createProject(
+        {
+            name,
+        },
+        user,
+        auditUser,
+    );
+    return project;
+};
+
+test('should return personal dashboard with membered projects', async () => {
+    // create project A with user 1
+    // create project B with user 1
+    const { body: user1 } = await loginUser('user1@test.com');
+    const projectA = await createProject('Project A', user1);
+    await createProject('Project B', user1);
+
+    // create project C with user 2
+    const { body: user2 } = await loginUser('user2@test.com');
+    const projectC = await createProject('Project C', user2);
+
+    // Add user 2 as a member of project A
+    await app.services.projectService.addAccess(
+        projectA.id,
+        [5],
```

Comment:
what is 5?

Reply:
That is the ID for project members. I guess I could add a comment here; will fix